smaps/src/main/java/com/google/smaps/Region.java

```diff
@@ -90,7 +90,8 @@
   abstract List<String> vmFlags();
 
   static Builder builder() {
-    return new AutoValue_Region.Builder();
+    // TODO(sophbohr22): Add these set initializations for all fields.
+    return new AutoValue_Region.Builder().setShmemHugePages(0).setHugePFNMap(0);
```

Comment:
Can you add newlines before each set?
So it would look like this:
..Builder()
      .setShmemHugePages(0)
      .setHugePFNMap(0);

Will the linter allow it?

Reply:
The linter won't allow it right now, but I think once we add more fields it will allow that formatting.